src/js/user-profile/components/FormList.jsx

```diff
@@ -0,0 +1,24 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import FormItem from './FormItem';
+
+class FormList extends React.Component {
+  render() {
+    const handleClick = this.props.handleClick;
+    const forms = this.props.savedForms;
+    return (
+      <div>
+        <h4 className="section-header">Saved applications</h4>
+        {forms.map((form, index) =>  <FormItem key={index} savedFormData={form} handleClick={handleClick}/>)}
```

Comment:
They `key` could be the form id, there's only one saved application per form.

Reply:
Great idea, will change.